<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QVTKOpenGLWidget" name="openGLWidget"/>
    </item>
    <item>
     <widget class="QGroupBox" name="groupBox">
      <property name="title">
       <string>3D Shapes</string>
      </property>
      <widget class="QPushButton" name="drawSphere_button">
       <property name="geometry">
        <rect>
         <x>40</x>
         <y>390</y>
         <width>256</width>
         <height>24</height>
        </rect>
       </property>
       <property name="text">
        <string>Draw</string>
       </property>
      </widget>
      <widget class="QWidget" name="layoutWidget">
       <property name="geometry">
        <rect>
         <x>40</x>
         <y>40</y>
         <width>271</width>
         <height>22</height>
        </rect>
       </property>
       <layout class="QHBoxLayout" name="horizontalLayout_3">
        <item>
         <widget class="QRadioButton" name="sphere_radioBtn">
          <property name="text">
           <string>Sphere</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QRadioButton" name="cube_radioBtn">
          <property name="text">
           <string>Cube</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QRadioButton" name="cone_radioBtn">
          <property name="text">
           <string>Cone</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QRadioButton" name="cylinder_radioBtn">
          <property name="text">
           <string>Cylinder</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QGroupBox" name="groupBox_2">
       <property name="geometry">
        <rect>
         <x>40</x>
         <y>80</y>
         <width>271</width>
         <height>111</height>
        </rect>
       </property>
       <property name="title">
        <string>Coordinates</string>
       </property>
       <widget class="QWidget" name="layoutWidget">
        <property name="geometry">
         <rect>
          <x>20</x>
          <y>20</y>
          <width>231</width>
          <height>86</height>
         </rect>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout">
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_4">
           <item>
            <widget class="QLabel" name="xPosition_lbl">
             <property name="text">
              <string>X</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QDoubleSpinBox" name="xPosition_spinBox">
             <property name="minimum">
              <double>-99.989999999999995</double>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_5">
           <item>
            <widget class="QLabel" name="yPosition_lbl">
             <property name="text">
              <string>Y</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QDoubleSpinBox" name="yPosition_spinBox">
             <property name="minimum">
              <double>-99.989999999999995</double>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_6">
           <item>
            <widget class="QLabel" name="zPosition_lbl">
             <property name="text">
              <string>Z</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QDoubleSpinBox" name="zPosition_spinBox">
             <property name="minimum">
              <double>-99.989999999999995</double>
             </property>
            </widget>
           </item>
          </layout>
         </item>
        </layout>
       </widget>
      </widget>
      <widget class="QStackedWidget" name="stackedWidget">
       <property name="geometry">
        <rect>
         <x>120</x>
         <y>220</y>
         <width>120</width>
         <height>80</height>
        </rect>
       </property>
       <property name="currentIndex">
        <number>0</number>
       </property>
       <widget class="QWidget" name="Sphere">
        <widget class="QWidget" name="layoutWidget">
         <property name="geometry">
          <rect>
           <x>0</x>
           <y>20</y>
           <width>91</width>
           <height>24</height>
          </rect>
         </property>
         <layout class="QHBoxLayout" name="horizontalLayout_2">
          <item>
           <widget class="QLabel" name="sphereRadius_lbl">
            <property name="text">
             <string>sphere</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QDoubleSpinBox" name="sphereRadius_spinBox"/>
          </item>
         </layout>
        </widget>
       </widget>
       <widget class="QWidget" name="Cylinder">
        <widget class="QWidget" name="layoutWidget">
         <property name="geometry">
          <rect>
           <x>10</x>
           <y>20</y>
           <width>105</width>
           <height>24</height>
          </rect>
         </property>
         <layout class="QHBoxLayout" name="horizontalLayout_7">
          <item>
           <widget class="QLabel" name="label">
            <property name="text">
             <string>ff</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QDoubleSpinBox" name="doubleSpinBox"/>
          </item>
         </layout>
        </widget>
       </widget>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QVTKOpenGLWidget</class>
   <extends>QOpenGLWidget</extends>
   <header>qvtkopenglwidget.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
